readme.md

```diff
@@ -53,7 +53,71 @@ Directory names should be lowercase. Filenames should be singular.
 *todo: evaluate common prefix like `pagelib` and BEM naming.*
 
 ## Development setup and workflow
-*Coming soon.*
+The Android and iOS Wikipedia apps make use of the `wikimedia-page-library`. At some point their respective build phases invoke `npm install` which causes a *published* version of this library to be retrieved and used in their respective builds. 
```

Comment:
I've heard people complain that an 80 character width is outdated and I often find it constraining myself. However, it's so useful to open a class under test and the test itself (or another reference) side by side! It looks like this on my computer without any OS chrome:

![screenshot from 2017-05-08 17-55-45](https://cloud.githubusercontent.com/assets/1071806/25830084/bd22e524-3417-11e7-8bde-f508a98e933d.png)

That's barely accommodating two 80 character widths on a 3840px (effectively 1920px) display. I wouldn't mind compromising to 100 characters no questions asked but I was curious if you would describe how you're coding. Do you just flip between files or do you have an ultra wide monitor?